test/function/examples/example_checker_bundle/files/result_file_ok.xqar

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<CheckerResults version="1.0.0">
+
+  <CheckerBundle build_date="" description="" name="DemoCheckerBundle" summary="Found 3 issues" version="">
+    <Checker checkerId="exampleChecker" description="This is a description" status="completed" summary="">
+      <Issue description="This is an information from the demo usecase" issueId="0" level="3" ruleUID=""/>
```

Comment:
Should a non-exisiting ruleUID not be handled via eliding the attribute rather than supplying an empty string?

More generally I think going forward the schema should be tightened down to specify required attributes (currently all are optional) and more specific types for the attributes, where useful (which would e.g. disallow empty strings in places where IDs are required). This is less of a comment on the current PR, but again more for the forward-looking architectural discussions.